test/models/custom_hint_test.rb

```diff
@@ -106,7 +67,7 @@ class CustomHintTest < ActiveSupport::TestCase
 
   test 'adds fingerprint if not present' do
     VCR.use_cassette('custom hint blank fingerprints', allow_playback_repeats: true) do
-      url = 'https://www.dropbox.com/s/3dgxge8b14ht0c3/custom%20hint%20for%20test%20suite.csv?dl=0'
+      url = 'https://www.dropbox.com/s/3dgxge8b14ht0c3/custom%20hint%20for%20test%20suite.csv?dl=1'
```

Comment:
I'm trying to understand whether this change needs a corresponding change in a cassette that I'm not seeing?

Reply:
The cassettes were looking for dl=1 but we removed that code that automatically changed that from dl=0 so I just updated the test instead of the cassette.

Reply:
Ah, gotcha